config/routes.rb

```diff
@@ -76,6 +76,8 @@
     get 'limited', to: 'example#limited', as: :limited
     get 'status', to: 'admin#status'
 
+    resources :ppiu, only: :index
```

Comment:
even though there's only one endpoint in the EVSS service would this be more self documenting as `/v0/ppiu/payment_information` ?

Reply:
Naming things is difficult. I ended up not doing it because I think `PPIU` stand for `Payment Processing something something`(?). But explicity is always good.